### PR TITLE
Fix pipeline

### PR DESCRIPTION
**Pull request by wind57**

### CODE REVIEW

File: spring-cloud-kubernetes-integration-tests/kind-config.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ containerdConfigPatches:
 # 1 control plane node and 3 workers
 nodes:
   - role: control-plane
-    image: kindest/node:v1.14.10@sha256:6cd43ff41ae9f02bb46c8f455d5323819aec858b99534a290517ebc181b443c6
```
Comment (wind57) [inline spring-cloud-kubernetes-integration-tests/kind-config.yaml:13]: upgrade the depen

File: spring-cloud-kubernetes-integration-tests/run.sh [hunk 2]
```diff
@@ -73,7 +73,7 @@ install_kind_release() {
 	elif [[ "$OSTYPE" == "win32" ]]; then
         KIND_BINARY_URL="https://github.com/kubernetes-sigs/kind/releases/download/${VERSION}/kind-windows-amd64"
 	else
-        echo "Uknown OS, using linux binary"
+        echo "Unknown OS, using linux binary"
```
Comment (wind57) [inline spring-cloud-kubernetes-integration-tests/run.sh:76]: typo

File: spring-cloud-kubernetes-integration-tests/run.sh [hunk 3]
```diff
@@ -87,7 +87,7 @@ main() {
     cd $CURRENT_DIR
 
     #TODO what happens if cluster is already there????
-    "${KIND}" create cluster --config=kind-config.yaml --loglevel=debug
```
Comment (wind57) [inline spring-cloud-kubernetes-integration-tests/run.sh:90]: `--loglevel=debug` is deprecated; instead `-v 2147483647`; its like saying "maximum logging output"

File: spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-core-k8s-client-it/src/test/resources/spring-cloud-kubernetes-core-k8s-client-it-ingress.yaml [hunk 4]
```diff
@@ -1,14 +1,18 @@
-apiVersion: networking.k8s.io/v1beta1
```
Comment (wind57) [inline spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-core-k8s-client-it/src/test/resources/spring-cloud-kubernetes-core-k8s-client-it-ingress.yaml:1]: no beta

File: spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-core-k8s-client-it/src/test/java/org/springframework/cloud/kubernetes/core/k8s/it/ActuatorEndpointIT.java [hunk 5]
```diff
@@ -107,8 +107,8 @@ private static V1Service getCoreK8sClientItService() throws Exception {
 		return service;
 	}
 
-	private static NetworkingV1beta1Ingress getCoreK8sClientItIngress() throws Exception {
-		NetworkingV1beta1Ingress ingress = (NetworkingV1beta1Ingress) k8SUtils
+	private static V1Ingress getCoreK8sClientItIngress() throws Exception {
```
Comment (wind57) [inline spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-core-k8s-client-it/src/test/java/org/springframework/cloud/kubernetes/core/k8s/it/ActuatorEndpointIT.java:110]: no beta

**Comment by wind57** [comment]
@ryanjbaxter ready to review

File: spring-cloud-kubernetes-integration-tests/run.sh [hunk 6]
```diff
@@ -87,7 +87,7 @@ main() {
...
+    "${KIND}" create cluster --config=kind-config.yaml -v 2147483647
```
Comment (ryanjbaxter) [inline spring-cloud-kubernetes-integration-tests/run.sh:90]: What is the `-v` param?

Comment (wind57) [reply spring-cloud-kubernetes-integration-tests/run.sh:90]: `--loglevel=debug` is deprecated. `-v 2147483647` is like saying "max output possible", sort of like trace

**Pull request merged**